src/main/java/edu/cornell/kfs/fp/service/impl/RecurringDisbursementVoucherDocumentServiceImpl.java

```diff
@@ -272,6 +275,24 @@ private void saveDisbursementVouchers(List<DisbursementVoucherDocument> dvs, Rec
             }
         }
     }
+    
+    private String buildDVExplanation(RecurringDisbursementVoucherDocument recurringDV) {
+        String dvExplanation = CuFPConstants.RecurringDisbursementVoucherDocumentConstants.RECURRING_DV_EXPLANATION_TO_DV_NOTE_STARTER
+                + recurringDV.getDocumentNumber() + KFSConstants.DELIMITER + KFSConstants.BLANK_SPACE + recurringDV.getDocumentHeader().getExplanation();
```

Comment:
Do you think it would be worth while adding some logic to only include the coma and space only if the recurring DV has an explanation to append?

I don't have a strong opinion on this, but when I start appending a number of String together, I think using a StringBuilder is more efficient, and slightly easier to read.